_posts/2016-xx-xx-release-0.16.md

```diff
@@ -126,42 +126,49 @@ MDAnalysis typically reads trajectories from files on-demand, so that it can eff
 
 ```python
+import numpy as np
 from MDAnalysis import Universe
-from MDAnalysisTests.datafiles import DCD, PDB_small
+from MDAnalysisTests.datafiles import DCD, PSF
 from MDAnalysis.coordinates.memory import MemoryReader
 
 # Create a Universe using a DCD reader
-universe = Universe(PDB_small, DCD)
+universe = Universe(PSF, DCD)
 
-# Extract coordinates
-coordinates = universe.trajectory.timeseries()
+# Create a numpy array with random coordinates (100 frames) for the same topology
+coordinates = np.random.uniform(size=(universe.atoms.n_atoms, 100, 3)).cumsum(0)
 
 # Create a new Universe directly from these coordinates
-# using the MemoryReader
-universe2 = Universe(PDB_small, coordinates,
-                     format=MemoryReader)
+universe2 = Universe(PSF, coordinates, format=MemoryReader)
 ```
 
-The MemoryReader will work just as any other reader. In particular, you can iterate over it as usual, or use the `.timeseries()` method to retrieve a reference to the raw array in any format:
+The MemoryReader will work just as any other reader. In particular, you can iterate over it as usual, or use the `.timeseries()` method to retrieve a reference to the raw array in any order of the dimensions ('fac' means ('frames','atoms','coordinates')):
 
 ```python
 coordinates_fac = universe2.trajectory.timeseries(format='fac')
 ```
 
-Certain operations can be speeded up by moving a trajectory to memory. To facilitate this operation, the constructor of `Universe` takes an `in_memory` flag which will automatically convert any trajectory to a MemoryReader:
+Certain operations can be speeded up by moving a trajectory to memory, and we have therefore
+added functionality to directly transfer any existing trajectory to a MemoryReader using `Universe.transfer_to_memory`:
 
 ```python
-universe = Universe(PDB_small, DCD, in_memory=True)
+universe = Universe(PSF, DCD)
+universe.transfer_to_memory()     # Switches to a MemoryReader representation
+```
+
+You can also do this directly upon construction of a Universe, by using the `in_memory` flag:
+
+```python
+universe = Universe(PSF, DCD, in_memory=True)
 ```
 
 Likewise, the `rms_fit_trj` function in the analysis/align.py module also has an `in_memory` flag, allowing it to do in-place alignments in memory.
 
 
 ## Incorporation of the ENCORE ensemble similarity library
 
-The ENCORE ensemble similarity library has been integrated with MDAnalysis. It implements a variety of techniques for calculating similarities between structural ensembles (trajectories), as described in this publication:
+The **ENCORE** ensemble similarity library has been integrated with MDAnalysis as [MDAnalysis.analysis.encore](http://www.mdanalysis.org/mdanalysis/documentation_pages/analysis/encore.html). It implements a variety of techniques for calculating similarities between structural ensembles (trajectories), as described in this publication:
```

Comment:
Sorry, I wasn't really clear where the docs for releases live: On PyPi but we have redirects, so the link should be

http://docs.mdanalysis.org/documentation_pages/analysis/encore.html

(assuming that PR #1047 gets merged.)